dist/api_data.py

```diff
@@ -771,6 +771,9 @@ def __cmp__(self, other):
     Config('remove_files', 'true', r'''
         should the underlying files be removed?''',
         type='boolean'),
+    Config('lock_wait', 'true', r'''
+        wait for locks, if false will return \c EBUSY instead of blocking''',
```

Comment:
You're right, we certainly can return `EBUSY` without this change.  I'll try to clarify the wording...